react-elements/src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const word = React.createElement(
+  'h1',
+  [null],
+  ['Hello, React!']
```

Comment:
In that part of the React docs, the square brackets means "optional", so here, you can remove the brackets.